src/system/Form/Select.test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+ 
+/**
+  * Internal dependencies
+  */
+import { Select } from './Select';
+ 
+const defaultProps = {};
+ 
+describe( '<Select />', () => {
+    it( 'renders the Select component with the default placeholder', async () => {
+        const { container } = render( <Select { ...defaultProps } /> );
+
+        const placeholder = screen.getByText( content, element => {
```

Comment:
I think the error is on this line: `content` is not defined. It needs a string in order to search an element by text.

Reply:
Fixed 👍 I was missing some parentheses, but I ended up scrapping this code and just searching for the text itself.